fitnesse/resources/bootstrap-plus/js/bootstrap-plus.js

```diff
@@ -216,5 +222,109 @@ $( document ).ready(function() {
                 $('#autoSave-switch').addClass('fa-toggle-on');
             }
         }
+    // Distinction made between different types of pages so it doesn't apply to all parents
+    // Add hover function to suite page to show add tag button
+    $('.suite').parent().hover(
+        function() {
+            $(this).find('.addTag:first').css("visibility", "visible");
+        }, function() {
+            $(this).find('.addTag:first').css("visibility", "hidden");
+        }
+    );
+    // Add hover function to static page to show add tag button
+    $('.static').parent().hover(
+        function() {
+            $(this).find('.addTag:first').css("visibility", "visible");
+        }, function() {
+            $(this).find('.addTag:first').css("visibility", "hidden");
+        }
+    );
+    // Add hover function to test page to show add tag button
+    $('.test').parent().hover(
+        function() {
+            $(this).find('.addTag:first').css("visibility", "visible");
+        }, function() {
+            $(this).find('.addTag:first').css("visibility", "hidden");
+        }
+    );
```

Comment:
Ik zou de functie voor het toggelen 1x in een losse functie zetten en die 3x aanroepen, ipv 3x dezelfde logica te gebruiken.

Reply:
Hover functie geoptimaliseerd. Zie commit (en indien nodig de screenshot hieronder).

![image](https://user-images.githubusercontent.com/38794020/72602796-215d9180-3918-11ea-90dd-e56a2e7537a4.png)